platforms/minestom/src/main/java/com/dfsek/terra/minestom/world/MinestomChunkGeneratorWrapper.java

```diff
@@ -0,0 +1,56 @@
+package com.dfsek.terra.minestom.world;
+
+import com.dfsek.terra.api.world.chunk.generation.ChunkGenerator;
+
+import com.dfsek.terra.api.world.chunk.generation.stage.GenerationStage;
+import com.dfsek.terra.minestom.api.filter.ChunkFilter;
+import com.dfsek.terra.minestom.chunk.CachedChunk;
+import com.dfsek.terra.minestom.chunk.GeneratedChunkCache;
+
+import net.minestom.server.coordinate.Point;
+import net.minestom.server.instance.generator.GenerationUnit;
+import net.minestom.server.instance.generator.Generator;
+import org.jetbrains.annotations.NotNull;
+
+
+public class MinestomChunkGeneratorWrapper implements Generator {
```

Comment:
Why is this not implementing `GeneratorWrapper`?

eg. https://github.com/PolyhedralDev/Terra/blob/0a952cff4c3eea8bc0a49205643e0c225d12304c/platforms/bukkit/common/src/main/java/com/dfsek/terra/bukkit/generator/BukkitChunkGeneratorWrapper.java#L40

Reply:
fixed